python/ray/tests/test_metrics.py

```diff
@@ -19,6 +19,13 @@
 
 
 def test_worker_stats(shutdown_only):
+    if _WIN32:
+        # Make sure the previous agent has indeed gone away
+        # Sometimes the Windows tasks claim they are dead before they
+        # actually release all the resources. This manifests as a failure
+        # to start up the agent.
+        time.sleep(5)
+
```

Comment:
This is tricky to test since it is the first test in the file. I needed to run a series of tests to get the agent to fail to start with a "Windows Error 10013: An attempt was made to access a socket in a way forbidden" in the dashboard_agent.txt log. I am not sure this is the best way to prevent the problem. If this helps make the test less flaky, we should think about a more systematic way to ensure a pause after `ray stop`.